### DISCUSSION TITLE
We detected that your site is not verifying reCAPTCHA solutions.

### DISCUSSION THREAD

**Discussion by Adr0it** [emailjs-com/help#35] [Q&A]
I had reCAPTCHA working fine for a few months using EmailJS to verify, but now, it's saying that my site is not verifying "reCAPTCHA solutions". I didn't really change anything, so I don't know if this is a new update or glitch. If this is on my end, I don't really know where to start since everything was working fine before.

**Comment by xr0master** [comment]
You should test your application and make sure that the keys are correct. It is possible that you just have a few users, and there were a couple of cases when someone activated the captcha but did not send the request at the end.

If the keys are correct and the captcha is enabled in your template, then I think you can just ignore this warning.

**Reply by Adr0it** [reply]
Oh, I think you're right. I was testing the captcha without submitting the entire form.
